src/Orleans.Transactions/Abstractions/ITransactionalStateStorage.cs

```diff
@@ -1,7 +1,11 @@
-﻿using Orleans.Concurrency;
-using System;
+﻿using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
+using Orleans.CodeGeneration;
+using Orleans.Concurrency;
+
+[assembly: GenerateSerializer(typeof(Orleans.Transactions.Abstractions.PendingTransactionState<>))]
+[assembly: GenerateSerializer(typeof(Orleans.Transactions.Abstractions.TransactionalStorageLoadResponse<>))]
```

Comment:
I prefer assembly-level attributes live in a file called AssemblyInfo.cs which only contains assembly-level metadata